website/docs/docs-section/appendix/savings-comparisons.mdx

```diff
@@ -0,0 +1,186 @@
+---
+id: savings-comparisons
+title: Costs & Savings (Beta)
+sidebar_label: Costs & Savings (Beta)
+---
+
+import { FiatLabel } from "../../../src/components/FiatLabel";
+import { BoxTag } from "../../../src/components/BoxTag";
+import { useFetchGasCost } from "../../../src/hooks";
+
+Are you a new user of TradeTrust or looking at deploying a new Token Registry (Transferable Documents)? If you are, we'd totally suggest that you try out our new V4 beta version!
+
```

Comment:
sorry got abit confused on versions, what's in v4 and what to look out for? we now have v3 and v4 in betas? 🤔

Reply:
ah ok so there's `OpenAttestation V3 (Beta)` and `Token Registry V4 (Beta)`